ui/src/app/pages/admin/workspace/admin-workspace.tsx

```diff
@@ -36,166 +35,156 @@ interface Props
-
-    // cloud storage traffic isn't always available (e.g. for a deleted workspace) so we need to allow for that
-    workspaceAdminApi()
-      .getCloudStorageTraffic(ns)
-      .then((cloudStorageTraffic) => this.setState({ cloudStorageTraffic }))
-      .catch(() => {});
-
-    // runtimes and user apps calls have error modes which cause them to be slow, so execute these in parallel
-    // to the main admin view call
-
-    workspaceAdminApi()
-      .adminListRuntimes(ns)
-      .then((runtimes) => this.setState({ runtimes }))
-      .catch(this.handleDataLoadError);
-
-    workspaceAdminApi()
-      .adminListUserAppsInWorkspace(ns)
-      .then((userApps) => this.setState({ userApps }))
-      .catch(this.handleDataLoadError);
-
-    workspaceAdminApi()
-      .getWorkspaceAdminView(ns)
-      .then((workspaceDetails) => this.setState({ workspaceDetails }))
-      .catch(this.handleDataLoadError)
-      .finally(() => this.setState({ loadingWorkspace: false }));
-  }
-
-  async populateWorkspaceDetails() {
-    const { ns } = this.props.match.params;
-    this.setState({ loadingWorkspace: true });
-
-    await workspaceAdminApi()
-      .getWorkspaceAdminView(ns)
-      .then((workspaceDetails) => this.setState({ workspaceDetails }))
-      .catch(this.handleDataLoadError)
-      .finally(() => this.setState({ loadingWorkspace: false }));
-  }
-
-  render() {
-    const { profile } = profileStore.get();
-    const {
-      cloudStorageTraffic,
-      loadingWorkspace,
-      dataLoadError,
-      workspaceDetails: { collaborators, resources, workspace, activeStatus },
-      runtimes,
-      userApps,
-    } = this.state;
-    const { workspaceObjects, cloudStorage } = resources || {};
-    const { researchPurpose } = workspace || {};
-    return (
-      <div style={{ margin: '1.5rem' }}>
-        {dataLoadError && (
-          <ErrorDiv>
-            Error loading data. Please refresh the page or contact the
-            development team.
-          </ErrorDiv>
-        )}
-        {loadingWorkspace && <SpinnerOverlay />}
-        {workspace && (
-          <div>
-            {activeStatus === WorkspaceActiveStatus.ACTIVE && (
-              <AdminLockWorkspace
-                {...{ workspace }}
-                reload={async () =>
-                  await this.populateFederatedWorkspaceInformation()
-                }
-              />
-            )}
-            <BasicInformation
-              {...{ workspace, activeStatus }}
-              reload={async () => await this.populateWorkspaceDetails()}
+  const populateFederatedWorkspaceInformation = async () => {
+    const { ns } = props.match.params;
+    setLoadingWorkspace(true);
+
+    try {
+      const newCloudStorageTraffic =
+        await workspaceAdminApi().getCloudStorageTraffic(ns);
+      setCloudStorageTraffic(newCloudStorageTraffic);
+    } catch (error) {
+      console.log('Error loading cloud storage traffic: ', error);
```

Comment:
This had an empty catch before. Our linter does not like that. Using handleDataLoadError like the other loaders doesn't seem right though, because when that is set, it seems like the rest of the page is not shown. I believe that the empty block was to avoid that if we run into this case.

Reply:
That sounds right to me.  As I recall, this is an error-prone call of lesser importance